### PR TITLE
Issues styling differently a reusable component- props.primary not re…

### PR DESCRIPTION
**Pull request by 0750kosse**
…cognized

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -59,15 +71,21 @@ const reviews: Review[] = [
...
+padding-top: 4rem;
 ${mediaQueries("tablet")`
   flex-direction: row;
 `} `
 
-function App() {
+// const Wrapper = styled.div<InfoSectionProps>`
+// ${props => props.primary && css`
+// background:linear-gradient(90deg,rgba(120,128,145, 0.34) 40%,#EBEBEB 0%)
+//  `}`
```
Comment (t0nin0s) [inline src/App.tsx:86]: remove if not needed

File: src/App.tsx [hunk 2]
```diff
@@ -59,15 +71,21 @@ const reviews: Review[] = [
...
+
+const App: React.FC<InfoSectionProps> = ({ children, ...rest }) => {
```
Comment (t0nin0s) [inline src/App.tsx:88]: App type shouldnt be 'infoSectionProps'

File: src/components/AboutUs/index.tsx [hunk 3]
```diff
@@ -1,58 +1,63 @@
...
-const Container = styled.div`
+const Container = styled.div<InfoSectionProps>`
 display: flex;
 flex-direction: column;
-padding:2rem;
 letter-spacing: 2px;
 line-height: 1.8em;
 align-items: center;
+padding:2rem;
+background: ${props => props.primary ? "red" : "yellow"}
```
Comment (t0nin0s) [inline src/components/AboutUs/index.tsx:24]: change to the right colors

File: src/components/AboutUs/index.tsx [hunk 4]
```diff
@@ -1,58 +1,63 @@
 import React from 'react';
-import styled from 'styled-components'
+import styled, { css } from 'styled-components'
```
Comment (t0nin0s) [inline src/components/AboutUs/index.tsx:2]: css not used, remove it

File: src/components/AboutUs/index.tsx [hunk 5]
```diff
@@ -1,58 +1,63 @@
...
+color: black;
 ${mediaQueries("tablet")`
-font-size: 3rem`} 
-`
-const styledParagraph = styled.p`
-color: #666;
+font-size: 3rem`}
 `
+
+const StyledParagraph = styled.p<InfoSectionProps>``
```
Comment (t0nin0s) [inline src/components/AboutUs/index.tsx:48]: no need to create a styledParagraph if you are not going to style it

File: src/components/AboutUs/index.tsx [hunk 6]
```diff
@@ -1,58 +1,63 @@
...
 `
-
-export interface InfoSectionProps {
-  logo?: any;
-  header: string;
-  subheader: string;
-  content: string[];
-  footerLogo?: any;
-}
+const BookNowButton = styled(Button)``
```
Comment (t0nin0s) [inline src/components/AboutUs/index.tsx:56]: use just Button instead of creating a new component

File: src/components/AboutUs/index.tsx [hunk 7]
```diff
@@ -61,14 +66,14 @@ export default (props: InfoSectionProps) => {
       <Content >
         <Header> {header}</Header>
         <SubHeader>{subheader}</SubHeader>
-        {content.map((paragraph: string, index: number) =>
-          <p key={index}>{paragraph}</p>
+        {content!.map((paragraph: string, index: number) =>
+          <StyledParagraph key={index}>{paragraph}</StyledParagraph>
```
Comment (t0nin0s) [inline src/components/AboutUs/index.tsx:70]:
```suggestion
          <p key={index}>{paragraph}</p>
```

File: src/components/AboutUs/index.tsx [hunk 8]
```diff
@@ -61,14 +66,14 @@ export default (props: InfoSectionProps) => {
...
         )}
+        {cta && <BookNowButton>{ctaText}</BookNowButton>}
```
Comment (t0nin0s) [inline src/components/AboutUs/index.tsx:72]:
```suggestion
        {cta && <Button>{ctaText}</Button>}
```

File: src/components/AboutUs/index.tsx [hunk 9]
```diff
@@ -1,58 +1,63 @@
...
+background: ${props => props.primary ? "red" : "yellow"}
 `
 const LogoImage = styled.img`
 width: 25%;
 padding: 2rem;
-border-radius:60%;
+border-radius: 60%;
 `
-const Content = styled.div`
+const Content = styled.div<InfoSectionProps>`
```
Comment (t0nin0s) [inline src/components/AboutUs/index.tsx:31]: dont pass any props if you dont need them
```suggestion
const Content = styled.div`
```

**Review by 0750kosse — COMMENTED** [review]
Pls double-check? All ok now?

**Review by t0nin0s — APPROVED** [review]

**Pull request merged**